README.md

```diff
@@ -203,15 +208,21 @@ Your cryptvm now should boot automatically into your fully encrypted system.
 
 ### Troubleshooting
 
-If the keyserver should be unreachable for whatever reason, you will be dropped into the `(initramfs)`-shell after a few minutes. From here you will be able to unlock your encrypted partition manually using a password.
+#### Manual unlocking
+
+If the keyserver should be unreachable for whatever reason, you will be dropped into the `(initramfs)`-shell after a few minutes. From here you will be able to unlock your encrypted partition manually using a password. For this to work it is critical, that *CRYPTSETUP=y* has been set, before creating the initramfs image or else the cryptsetup won't be available in initramfs emergency shell.
 
 ```bash
-(initramfs) cryptsetup open /dev/sda1 encrypted-rootfs
-Enter passphrase for /dev/sda1:
+(initramfs) cryptsetup open /dev/sda5 sda5_crypt
+Enter passphrase for /dev/sda5:
 (initramfs) exit
 ```
 
-You will need to modify `sda0` and `encrypted-rootfs` accordingly.
+Maybe you will need to modify `sda5` and `sda5_crypt` accordingly.
+
+#### Issues on initial setup
+
+If something went wrong on the initial setup and you're locked out, you can use the backed up initrd.img.cryptbootbackup. In order to do that, you would hit the *e*-key on the entry in GRUB. In this editor you'd edit the last line `initrd /initrd.img-4.9.0-7-amd64` line to say `initrd initrd.img.cryptbootbackup` and press F10. Since the backup will only be created by the script, this will probably not work if the setup should break later on, after new kernel versions have been installed.
```

Comment:
Debianoid (not sure about others) systems should also have a initrd.img.old to recover. Not sure if this comes with each kernel upgrade or only with dist-upgrades. 

I also added a manual backup step to the how to.

Reply:
```
lrwxrwxrwx   1 root root   29 Jul 17 14:27 initrd.img -> boot/initrd.img-4.9.0-7-amd64
lrwxrwxrwx   1 root root   29 May 30 19:33 initrd.img.old -> boot/initrd.img-4.9.0-6-amd64
```
It looks like those are symlinks on different kernel package versions. I think someone should be able to boot an older kernel from GRUB, if they have previously upgraded.

Reply:
I will add this as another fallback mechanism.